addon/services/unified-event-handler.js

```diff
@@ -47,6 +47,11 @@ export default Ember.Service.extend(Ember.Evented, {
     this._throttledEventTimers = [];
   },
 
+  isFastBoot: Ember.computed(function() {
+    const fastbootService = Ember.getOwner(this).lookup('service:fastboot');
```

Comment:
Pardon my ignorance, but is this the recommended way to check if you're in FastBoot? Seems kind of roundabout, but I haven't done much with FastBoot.

Reply:
Yes, I believe this is the recommended way to check for fastboot:
https://www.ember-fastboot.com/docs/addon-author-guide#accessing-the-fastboot-service

I am willing to change it if you feel diff.

Reply:
Seems good to me then. Thanks for the reference.